tests/fixtures/rules/no-not-data-props-in-kit-pages/valid/test01-input.svelte

```diff
@@ -0,0 +1,6 @@
+<script>
+  export let data
+  export let errors
```

Comment:
I saw this and noticed that `errors` is also allowed.
So I think it would be better to change the rule name (`no-not-data-props-in-kit-pages`) from what I suggested 😓.

Do you have any good ideas?
For example, how about a name like `valid-prop-names-in-kit-pages`?

Reply:
I changed the rule name according to your suggestion!